spec/queue_test.js

```diff
@@ -0,0 +1,31 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Stack from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
+'use strict'
+
+// it('exists', () => {
```

Comment:
Also you forgot your group name don't forget to write it out next time for your next reviewers. Thanks!